src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -300,7 +300,7 @@ internal override NamespaceExtent Extent
             // NOTE: a name maps into values collection containing types only instead of allocating another
             // NOTE: array of NamedTypeSymbol[] we downcast the array to ImmutableArray<NamedTypeSymbol>
 
-            var builder = new NameToSymbolMapBuilder(_mergedDeclaration.Children.Length);
+            using var builder = new NameToSymbolMapBuilder();
```

Comment:
>using [](http://example.com/codeflow?start=12&length=5)

Please get rid of the ```using```. We prefer to avoid using disposable pattern to manage pooled objects.

Reply:
done.